### PR TITLE
Update configure-liveness-readiness-startup-probes.md

### PR DESCRIPTION
**Pull request by YngwieWang**
Add a "TCP" to synchronize with the English version.

### CODE REVIEW

**Comment by YngwieWang** [comment]
I have signed the CLA

**Comment by YngwieWang** [comment]
Sorry, I thought I've signed the CLA yesterday, but I didn't finished it correctly.
Just now, I have already done that.

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -503,7 +503,7 @@ to resolve it.
 -->
 对于 HTTP 探测，kubelet 发送一个 HTTP 请求到指定的路径和端口来执行检测。除非 `httpGet` 中的 `host` 字段设置了，否则 kubelet 默认是给 Pod 的 IP 地址发送探测。如果 `scheme` 字段设置为了 `HTTPS`，kubelet 会跳过证书验证发送 HTTPS 请求。大多数情况下，不需要设置`host` 字段。这里有个需要设置 `host` 字段的场景，假设容器监听 127.0.0.1，并且 Pod 的 `hostNetwork` 字段设置为了 `true`。那么 `httpGet` 中的 `host` 字段应该设置为 127.0.0.1。可能更常见的情况是如果 Pod 依赖虚拟主机，你不应该设置 `host` 字段，而是应该在  `httpHeaders` 中设置 `Host`。
 
-对于一次探测，kubelet 在节点上（不是在 Pod 里面）建立探测连接，这意味着你不能在 `host` 参数上配置 service name，因为 kubelet 不能解析 service name。
+对于一次 TCP 探测，kubelet 在节点上（不是在 Pod 里面）建立探测连接，这意味着你不能在 `host` 参数上配置 service name，因为 kubelet 不能解析 service name。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md]: 确定“探测”是“TCP探测”

Comment (YngwieWang) [reply content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md]: 英文文档是 " For a TCP probe “

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Comment (tengqm) [reply content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md]: In that case, please change line 500 accordingly.

Comment (YngwieWang) [reply content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md]: done

Comment (YngwieWang) [reply content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md]: OK, thank you. I've finished it.

**Pull request merged**